test/libsolidity/semanticTests/externalContracts/prbmath_unsigned.sol

```diff
@@ -50,7 +50,7 @@ contract test {
 // compileViaYul: also
 // ----
 // constructor()
-// gas irOptimized: 1778342
+// gas irOptimized: 1778102
```

Comment:
This is rather unexpected... probably a fluke due to the ``memoryguard`` preventing some rematerialization...